charts/grafana/tests/grafana-deployment_test.yaml

```diff
@@ -7,3 +7,12 @@ tests:
     asserts:
       - isKind:
           of: Deployment
+  - it: should work with ssl mode
+    set:
+      global:
```

Comment:
@danielhoherd ☝️

Reply:
Globals are broken with the version of helm-unittest that we are using. There is a ticket open to move to a newer version (https://github.com/astronomer/issues/issues/2701) or we could switch to using pytest like FOSS airflow chart does (https://github.com/astronomer/airflow/blob/master/chart/tests/test_dags_persistent_volume_claim.py)

Reply:
yeah it's a good idea, we should invest some time since we are use a lot of helm-template logic.